### PR TITLE
fix: US138562 - Adding Enumeration to T/F, multiple-choice, multi-select

### PR DESCRIPTION
**Pull request by tdevos-d2l**
[US138562](https://rally1.rallydev.com/#/?detail=/userstory/632750608707&fdp=true): [Consistent Evaluation][Quizzes][UI][Questions]  Update existing d2l-questions-multi-select and hook up component to consume updated API

- Adding enumeration to
  - True/False
  - Multiple Choice
  - Multi-Select
- Also including a fix to Multi-Select which will not show the correct answer arrows when the question is fully correct. This is how it is done in Legacy
- Added v-diff tests for enumeration
- Updated aXe tests
- Updated unit tests
- Updated demo pages

### CODE REVIEW

File: components/helpers/htmlHelper.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ export function renderTextAsHTML(userHTML) {
 		.d2l-questions-html-block-no-tag-fixer {
 			margin: 0;
 			padding: 0;
+			line-height: 1.47;
```
Comment (tdevos-d2l) [inline components/helpers/htmlHelper.js:13]: This is added to line up html-rendered with non-html html-blocks. With out standardizing the `line-height` we get text not lining up with the enumerations
https://github.com/BrightspaceUI/core/blob/b92b16cfcf21ab63a03b152bb2939fe04d96243a/components/html-block/html-block.js#L12

**Comment by tdevos-d2l** [comment]
Waiting for a backend change before merging this in

File: components/d2l-questions-multi-select.js [hunk 2]
```diff
@@ -90,6 +97,9 @@ class D2lQuestionsMultiSelect extends SkeletonMixin(LitElement) {
 		correctChoiceHrefs = await this._loadCorrectChoices(candidateResponse);
 
 		const choices = await Promise.all(candidateResponse?.entities.map(async choice => {
+			if (!choice?.hasClass(Classes.questions.correctResponse)) {
```
Comment (tdevos-d2l) [inline components/d2l-questions-multi-select.js:100]: This checks to see if there are any incorrect responses in a question

File: components/d2l-questions-multi-select-presentational.js [hunk 3]
```diff
@@ -214,7 +250,7 @@ class D2lQuestionsMultiSelectPresentational extends SkeletonMixin(LocalizeQuesti
 	_renderReadonlyChoice(text, htmlText, checked, correctAnswer, correctResponse, accessibleDescription) {
 		return html`
 			<div class="d2l-questions-multi-select-row d2l-body-compact">
-				${correctAnswer ? html`<d2l-icon class="d2l-questions-multi-select-correct-answer-icon" icon="tier1:arrow-thin-right"></d2l-icon>` : ''}
+				${this.questionCorrect ? '' : correctAnswer ? html`<d2l-icon class="d2l-questions-multi-select-correct-answer-icon" icon="tier1:arrow-thin-right"></d2l-icon>` : ''}
```
Comment (tdevos-d2l) [inline components/d2l-questions-multi-select-presentational.js:253]: If the student gets no choices incorrect, we won't show the arrows pointing to the correct answers, as done in legacy

Comment (JamieTran) [reply components/d2l-questions-multi-select-presentational.js:253]: I don't see where `this.questionCorrect` defined 👀

**Review by JamieTran — APPROVED** [review]
lgtm! Awesome implementation of enumeration

File: components/helpers/htmlHelper.js (hunk 1 above)
Comment (maddiegabriel) [reply components/helpers/htmlHelper.js:13]: nice job

**Review by maddiegabriel — APPROVED** [review]
excellent stuff! :shipit:

**Pull request merged**